libraries/AP_HAL_ChibiOS/hwdef/GEPRCF745BTHD/hwdef.dat

```diff
@@ -0,0 +1,185 @@
+# hw definition file for processing by chibios_pins.py
+# for GEPRCF745-BT-HD hardware.
+# thanks to betaflight for pin information
+
+# MCU class and specific type
+MCU STM32F7xx STM32F745xx
+
+# board ID for firmware load
+APJ_BOARD_ID 1500
+
+# crystal frequency, setup to use external oscillator
+OSCILLATOR_HZ 8000000
+
+FLASH_SIZE_KB 1024
+
+# leave 2 sectors free
+FLASH_RESERVE_START_KB 96
+
+
+# only one I2C bus
+I2C_ORDER I2C1
+
+# I2C1 for baro
+PB8 I2C1_SCL I2C1
+PB9 I2C1_SDA I2C1
+
+# order of UARTs (and USB), 
+SERIAL_ORDER OTG1 USART1 USART2 USART3 UART4 EMPTY USART6 UART7 UART8
+
+# buzzer
+#PD2 TIM3_ETR TIM3 GPIO(77) ALARM
```

Comment:
should remove these as they are replicated below

Reply:
Already removed